### DISCUSSION TITLE
connecting to wallet

### DISCUSSION THREAD

**Discussion by voodoopant** [Chia-Network/chia-blockchain#3176] [General]
Hi !
I tried a force reload and I am stuck at connecting to wallet .
My first plotting was running .
What should I do so I don't lose the plotting progress ?
Thanks !

Windows 10 64 . Latest Chia Blockchain version .

**Comment by nbaird** [comment]
Open Task Manager and hit the v to the left of Chia. You will see a chia.exe process that is consuming high % of CPU and Memory. After that process completes you can close the GUI and re-open.

**Comment by voodoopant** [comment]
Closing node and server , how much time ?

**Comment by voodoopant** [comment]
Closing down node and server , how much time ?
